### DISCUSSION TITLE
Subtitles on android

### DISCUSSION THREAD

**Discussion by JoacimWall** [CommunityToolkit/Maui#882] [New Feature Discussions]
Hi anyone know if there are any plans for support subtitles on android in new MAUI mediaElement?

Now it's only works on iOS.   
//Regards Joacim

**Comment by JoacimWall** [comment]
This solved it for me right now. (tip from [jfversluis](https://github.com/jfversluis))
         
  .ConfigureMauiHandlers(handlers =>
            {
                MediaPlayerHandler.PropertyMapper.AppendToMapping("subtitleMapper", (handler, player) =>
                {
#if ANDROID
                    handler.PlatformView.GetFirstChildOfType<StyledPlayerView>()?.SetShowSubtitleButton(true);
                  
#endif
                });
                   });

**Comment by CliffAgius** [comment]
Closed and answered
